public/components/datasources/components/manage/associated_objects/associated_objects_details_flyout.tsx

```diff
@@ -242,6 +259,10 @@ export const AssociatedObjectsDetailsFlyout = ({
     );
   }, [tableColumns]);
 
+  useEffect(() => {
+    stopLoading();
```

Comment:
not sure why stop loading every time component initializes but if you are looking for stop last loading you could do

`useEffect(() => {
  return () => stopLoading(); // stop pulling when unmounts
}, [])`